self-host/postgres/20250416112843_alter_auth_sessions_table.sql

```diff
@@ -0,0 +1,9 @@
+-- migrate:up
+alter table public.auth_sessions
+add column own_team_id uuid;
+
+comment on column public.auth_sessions.own_team_id is 'team id of default team of the user';
+
+-- migrate:down
+alter table public.auth_sessions
+drop column if exists own_team_id;
```

Comment:
@anupcowkur see my other comment above.

Reply:
see my reply above discussing this

Reply:
I've tried to explain my concerns above. Can discuss more.

Reply:
will get rid of this